test/e2e/conf.js

```diff
@@ -13,11 +13,18 @@
 /* eslint no-console: 0 */
 require('./env').config();
 
+const jasmineReporters = require('jasmine-reporters');
+
 var config = {
   framework: 'jasmine2',
   specs: ['specs/*.js'],
   restartBrowserBetweenTests: false,
   onPrepare: function () {
+    jasmine.getEnv().addReporter(new jasmineReporters.JUnitXmlReporter({
```

Comment:
just curious, unit test uses karma as well but doesn't need dependency `jasmine-reporters`. why would e2e need?

Reply:
e2e tests use protractor, not karma and protractor config needs this dependency.

Reply:
ah, I see.. 🤦